projects.md

```diff
@@ -26,8 +25,8 @@ permalink: /projects/
 
 ## Mobile Apps
 
-* [Android mobile app](https://github.com/pebble-dev/android-store-app)
-* iOS mobile app *(source link coming soon)*
+* [Official Pebble Android app](https://www.apkmirror.com/apk/pebble-technology-corp/pebble/pebble-4-4-2-1405-62d45d7d7-endframe-release/pebble-4-4-2-1405-62d45d7d7-endframe-android-apk-download/) *(For use with Rebble Web Services. Site unaffiliated with Rebble, go to rebble.io/howto for full installation and setup instructions)*
```

Comment:
why not make this a link? :bowtie: 💭 🔗

Reply:
Great idea! I should have done that to begin with. I'll make that change when I have the chance.

Reply:
Done!